src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -3171,6 +3179,11 @@ components:
         phsId:
           type: string
           description: PHS ID (DbGap Phenotype Study Identifer) associated with dataset
+        selfHosted:
```

Comment:
were you going to name this something like `experimentalSelfHosted` and add some docs?  At least on creation?

Reply:
Yes! Was gonna do that this afternoon!

Reply:
Done.